src/mongo/Labeller.class.php

```diff
@@ -1,10 +1,20 @@
 <?php
+
+namespace Tripod\Mongo;
+
 /** @noinspection PhpIncludeInspection */
 require_once TRIPOD_DIR.'classes/Labeller.class.php';
```

Comment:
Unfortunately things go pretty haywire if it's removed.  I think a future issue is to ensure that Tripod is *actually* PSR compliant (so we're not dependent on requiring tripod.inc.php for it to work at all).